utils.py

```diff
@@ -240,10 +240,11 @@ def write_solution_to_file(solution_file_path: str,
     """
     num_cases = cases.num_cases
     num_bins = bins.num_bins
+    lowest_num_bin = bins.lowest_num_bin
     dx, dy, dz = effective_dimensions
     if num_bins > 1:
-        num_bin_used = sum([vars.bin_on[j].energy(sample)
-                            for j in range(num_bins)])
+        num_bin_used = lowest_num_bin + sum([vars.bin_on[j].energy(sample)
+                            for j in range(lowest_num_bin, num_bins)])
```

Comment:
Was the issue here that the `energy()` method was missing for vars indices less than `lowest_num_bin`? I'm probably missing something here, but why should the number of bins used be the sum of all energies?